### PR TITLE
Update tx service delegate methods

### PR DESCRIPTION
**Pull request by falvaradorodriguez**
Closes https://github.com/safe-global/safe-cli/issues/425

### CODE REVIEW

**Comment by falvaradorodriguez** [comment]
A new release of safe-eth-py is needed as the method that calculates the delegate signature in the current version is incorrect. Anyway, the task work in safe-cli would be ready.

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: tests/test_safe_tx_service_operator.py [hunk 1]
```diff
@@ -80,11 +80,21 @@ def test_add_delegate(self, add_delegate_mock: MagicMock):
         delegate_address = Account.create().address
         label = "Test"
         signer = list(safe_operator.accounts)[0]
+
+        expected_signature = safe_operator.safe_tx_service.create_delegate_message_hash(
```
[the remark this answers is not in the document]
Comment (falvaradorodriguez) [reply tests/test_safe_tx_service_operator.py:84]: ops! 😅

File: tests/test_safe_tx_service_operator.py [hunk 2]
```diff
@@ -80,11 +80,21 @@ def test_add_delegate(self, add_delegate_mock: MagicMock):
         delegate_address = Account.create().address
         label = "Test"
         signer = list(safe_operator.accounts)[0]
+
+        expected_hash = safe_operator.safe_tx_service.create_delegate_message_hash(
+            delegate_address
+        )
+        expected_signature_signed = signer.signHash(expected_hash)
```
Comment (moisses89) [inline tests/test_safe_tx_service_operator.py:87]:
```suggestion
        expected_signature = signer.signHash(expected_hash)
```
Looks like a bit redundant signature is supposed to be signed :)

**Review by moisses89 — APPROVED** [review]

File: tests/test_safe_tx_service_operator.py (hunk 2 above)
Comment (falvaradorodriguez) [reply tests/test_safe_tx_service_operator.py]: Yes, you are right. Updated https://github.com/safe-global/safe-cli/pull/427/commits/b8f86ad562d947a4f540256bfde2e90f578331ac

**Comment by falvaradorodriguez** [comment]
Added a blocked label to wait for a new version of safe-eth-py with the signature EIP712 fixed

**Review by moisses89 — APPROVED** [review]

**Pull request merged**